_shared/project/requirements/functests.in

```diff
@@ -4,6 +4,7 @@ pip-sync-faster
 httpretty
 pytest
 factory-boy
+pytest-factoryboy
```

Comment:
h and LMS both have their own hoary custom ways of integrating `factory_boy` into `pytest` (and they both do it differently as well). Let's just use <https://github.com/pytest-dev/pytest-factoryboy>, it's part of the `pytest-dev` organization on GitHub, we're already using in <https://github.com/hypothesis/gh-pr-upsert/>, and it integrates `factory_boy` into `pytest` in a really neat way (although it is pretty high magic).